journal/updated/09-chapter-8.adoc

```diff
@@ -195,50 +199,50 @@ and by me rather desirable that it may soon be experienced.
+On seventh-day we reached Worcester, and lodged at our kind friend Timothy Bevington`'s.
+Were at their meetings on first-day, where were Patience Brayton and her companion also.
 On second-day, the 24th, my dear companion left me, she being very desirous to get home;
-but I was not easy to leave Worcester so soon, and stayed more than a week longer,
+but I was not clear to leave Worcester so soon, and stayed more than a week longer,
 in which time I visited several of my acquaintance, had several satisfactory seasons,
 and was much favoured at some of their public meetings,
 so that I came away with sweet peace.
 
 On fourth-day morning I took an affectionate farewell of my dear friends,
 and was accompanied by George Beckett and Mary Bewley to Alcester, where they left me,
 and I lodged at the widow Stanley`'s,
-from which I was favoured with the company of an agreeable young woman, Mary Ford,
+from which place I was favoured with the company of an agreeable young woman, Mary Ford,
 to Jeffery Bevington`'s, at Eddington.
-I was at their meeting on fifth-day; and on sixth, at the meeting at Ormscott;
+I was at their meeting on fifth-day, and at the meeting at Ormscott on sixth,
 but the weather being severe, there were few Friends present;
 yet it was a time of deep travail and hard labour for the honest-hearted.
 On fourth-day I was at the meeting at Warwick, and on the following day reached Coventry,
 where I was joined by my husband.
-Attended their meeting on first-day, Timothy Bevington of Worcester, being there also,
-and we were owned by our Great Master, and employed in his service.
-On the 15th of second month we got safe home, and found our dear children well.
+Attended their meeting on first-day, Timothy Bevington of Worcester being there also,
+and we were acknowledged by our Great Master, and employed in His service.
+On the 15th of second month we got safely home, and found our dear children well.
 I had been absent on this occasion more than five months.
 
 [.asterism]
 '''
 
 +++[+++From her memorandums which come down to the fifth month, 1801,
 this journey appears to be the last which she performed
-with certificate from her Monthly Meeting,
-nevertheless she was almost continually engaged in the service of Truth,
+with a certificate from her Monthly Meeting.
+Nevertheless, she was almost continually engaged in the service of Truth,
 in attending meetings in various places, and often through much bodily infirmity,
 in the course of which her remarks are frequently weighty and edifying,
 of which the following are a few:]
 
 On the 17th of seventh month, 1735, I being very infirm and now at home,
 and having been often engaged in my good Master`'s service, and having a sure evidence,
-even the Truth bearing me witness that I am in the way of my duty to my great Lord,
+the Truth bearing me witness, that I am in the way of my duty to my great Lord,
```

Comment:
This should go back the way it was, or maybe add some long dashes on both sides. Because she is trying to describe WHAT is the sure evidence she had.  So her point is:

On the 17th of seventh month, 1735, I being very infirm and now at home, and having been often engaged in my good Master`'s service, and having a sure evidence **—** even the Truth bearing me witness that I am in the way of my duty to my great Lord **—** my spirits are borne up over all opposers and such as say hard things of me, and I have peace.